api/src/main/java/org/openmrs/module/pharmacyapi/api/model/stock/Lot.java

```diff
@@ -0,0 +1,102 @@
+/**
```

Comment:
We have here domain classes that we don't need anymore since that stock management is out of the scope